string-manipulation-1/src/is-vowel.js

```diff
@@ -6,9 +6,17 @@
 // say false if it is not
 
 function isVowel(char) {
-  if (char === 'a' || char === 'e' || char === 'i' || char === 'o' || char === 'u') {
-    return true;
-  } else if (char === 'A' || char === 'E' || char === 'I' || char === 'O' || char === 'U') {
+  char.toLowerCase();
```

Comment:
You need to assign the return of the `toLowerCase` method back to `char`, `toLowerCase` does not affect the original. Once you do that, you can remove the capital letters from your set of expressions in the condition.